changelogs/fragments/1059-iam_user-add-backoff-logic-handle-errors.yml

```
@@ -0,0 +1,4 @@
+minor_changes:
+- iam_user - wrap client with backoff logic to catch EntityTemporarilyUnmodifiable exception, clean up return struct (https://github.com/ansible-collections/community.aws/pull/1059).
```

```suggestion
- iam_user - wrap client with backoff logic to catch EntityTemporarilyUnmodifiable exception, clean up returned data structure (https://github.com/ansible-collections/community.aws/pull/1059).
```
This shorthand may not be familiar to all our users